packages/sitecore-jss-nextjs/src/components/Link.tsx

```diff
@@ -30,15 +30,18 @@ export const Link = forwardRef<HTMLAnchorElement, LinkProps>(
 
     if (
       !field ||
-      (!(field as LinkFieldValue).editable && !field.value && !(field as LinkFieldValue).href)
+      (!(field as LinkFieldValue).editable &&
+        !field.value &&
+        !(field as LinkFieldValue).href &&
+        !field.metadata)
     ) {
       return null;
     }
 
     const value = ((field as LinkFieldValue).href
       ? field
       : (field as LinkField).value) as LinkFieldValue;
-    const { href, querystring, anchor } = value;
+    const { href, querystring, anchor } = value || {};
```

Comment:
Was something changed with the _value_ type? Previously it wasn't required fallback to object

Reply:
some tests were failing - in case field is LinkFieldValue, href is empty and we don't have 'value' we want to fallback to react link and render empty component; however in that case the destructuring fails on that line

Reply:
@yavorsk It means that when field.value is empty it's not handled by HOC, isn't it?
this code shouldn't be executed if field.value is empty

Reply:
so the think in nextjs link is that we are not wrapping it with the hoc, bc react link would be double wrapped in that case. which withEmptyFieldEditincComponent is not that big of a problem i guess - just in some cases the same code would be executed twice for the same field. Should we wrap nextjs link also?

Reply:
@yavorsk I see, now it makes sense. Can you add a comment there may be? To explain why we fallback the value
Should it be the same problem for RichText component as well? Since we reference _value_ but it's not wrapped by HOC

Reply:
nextjs RichText is different in that, that all logic on what to return is made in the react rich text component, and we don't have Link's return null [here](https://github.com/Sitecore/jss/blob/b669ab5a913abcd7041b2240bb1c08ac50143c6d/packages/sitecore-jss-nextjs/src/components/Link.tsx#L38)

Reply:
@yavorsk As we discussed, for RichText it's not a problem, since _value_ is a string